jsonnet/dashboards/apps/ssl-exporter.libsonnet

```
@@ -0,0 +1,253 @@
+/*
+  Copyright 2020 The dNation Kubernetes Monitoring Authors. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local prometheus = grafana.prometheus;
+local row = grafana.row;
+local table = grafana.tablePanel;
+local statPanel = grafana.statPanel;
+
+{
+  grafanaDashboards+:: {
+
+    'ssl-exporter':
+      local totalUniqueCerts = statPanel.new(
+        title="Total Unique Certificates",
+        datasource="$datasource",
+        graphMode='none',
+      )
+      .addTarget(
+        prometheus.target(
+          format='table',
+          expr="count(max(ssl_cert_not_after{instance=~\"$instance\",job=~\"$job\"}) by (issuer_cn,serial_no))",
```

Please add space and remove slashes(not needed if you change `"` to `'`)
```suggestion
          expr='count(max(ssl_cert_not_after{instance=~"$instance", job=~"$job"}) by (issuer_cn,serial_no))',
```